hebiros_description/urdf/omnibase.xacro

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" ?>
+<robot name="rosie_test" xmlns:xacro="http://www.ros.org/wiki/xacro">
+  <xacro:include filename="$(find hebiros_description)/urdf/common_properties.xacro"/>
+  <xacro:include filename="$(find hebiros_description)/urdf/x_library.xacro"/>
+  <xacro:include filename="$(find hebiros_description)/urdf/x_series_common.xacro"/>
+
+  <link name="base_footprint"/>
+
+  <joint name="base_joint" type="fixed">
+    <parent link="base_footprint"/>
+    <child link="base_link" />
+    <origin xyz="0 0 0.010" rpy="0 0 0"/>
+  </joint>
+
+  <link name="base_link">
+    <visual>
+      <origin xyz="-0.1625 0.1775 0" rpy="1.5708 0 0"/>
+      <geometry>
+        <!--box size="0.50 0.50 0.10"/-->
+        <mesh filename="package://hebiros_description/meshes/omnibase_simple.stl" scale="0.001 0.001 0.001"/>
+      </geometry>
+      <material name="light_black"/>
+    </visual>
+
+    <collision>
+      <origin xyz="0 0 0" rpy="0 0 0"/>
+      <geometry>
+        <box size="0.50 0.50 0.1"/>
+      </geometry>
+    </collision>
+    
+  </link>
+<!---->
+  <joint name="module1_joint" type="fixed">
+    <parent link="base_link"/>
+    <child link="HEBI/Module1"/>
```

Comment:
We don't need the full "HEBI/Module1" name here; I'd just maintain consistency with the rest of the code in the file and have it named "module1" (or even change all the "module*" to "wheel*", or "wheel_actuator*" to clarify difference between omniwheel and actuator).